extension/src/test/suite/extension.test.ts

```diff
@@ -19,19 +19,14 @@ import { DvcReader } from '../../cli/dvc/reader'
 import expShowFixture from '../fixtures/expShow/base/output'
 import plotsDiffFixture from '../fixtures/plotsDiff/output'
 import * as Disposer from '../../util/disposable'
-import {
-  RegisteredCliCommands,
-  RegisteredCommands
-} from '../../commands/external'
+import { RegisteredCommands } from '../../commands/external'
```

Comment:
Saw these warnings when I ran `yarn lint`. I think they're leftover from #3035

Reply:
I followed up in #3039